app/views/admin/suppliers/_task.html.haml

```diff
@@ -3,12 +3,17 @@
     = task.framework.short_name
   %td.govuk-table__cell
     = task.due_on.to_s(:month_year)
-  %td.govuk-table__cell
-    - if task.latest_submission
+
+  - if task.latest_submission
```

Comment:
Does moving the `if` out here mean that we can end up with fewer `td.govuk-table__cell`s than there are `th.govuk-table__header`s?

Reply:
👍